prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1055,28 +1055,70 @@ static int rxr_ep_getopt(fid_t fid, int level, int optname, void *optval,
 	struct rxr_ep *rxr_ep = container_of(fid, struct rxr_ep,
 					     util_ep.ep_fid.fid);
 
-	if (level != FI_OPT_ENDPOINT || optname != FI_OPT_MIN_MULTI_RECV)
+	if (level != FI_OPT_ENDPOINT)
 		return -FI_ENOPROTOOPT;
 
-	*(size_t *)optval = rxr_ep->min_multi_recv_size;
-	*optlen = sizeof(size_t);
+	switch (optname) {
+	case FI_OPT_MIN_MULTI_RECV:
+		*(size_t *)optval = rxr_ep->min_multi_recv_size;
+		*optlen = sizeof(size_t);
+		break;
+	case FI_OPT_RNR_RETRY:
+		*(size_t *)optval = rxr_env.rnr_retry;
```

Comment:
rxr_env is a global and should not be used to store endpoint state

Reply:
Will have a new field in endpoint struct to store the RNR retry count.

Reply:
Fixed in new revision.